lib/fast_jsonapi/object_serializer.rb

```diff
@@ -232,8 +232,8 @@ def belongs_to(relationship_name, options = {}, &block)
         add_relationship(relationship)
       end
 
-      def meta(&block)
-        self.meta_to_serialize = block
+      def meta(meta_name = nil, &block)
```

Comment:
@attilahorvath why do we need `meta_name` here?

Reply:
That was changed to work similarly to [`set_id`](https://github.com/fast-jsonapi/fast_jsonapi/blob/master/lib/fast_jsonapi/object_serializer.rb#L179-L181).

This makes it possible to pass either a `proc` or a lambda or a `&:proc` shorthand to the `meta` method, instead of just a `do...end` block like before.

Here is the test to exercise this: https://github.com/fast-jsonapi/fast_jsonapi/pull/59/files#diff-2d575477b8a31f49298087fda89ac6a0R456

Reply:
Sorry missed it. Thanks for explaining this!